codalab/worker/main.py

```diff
@@ -140,6 +146,18 @@ def parse_args():
     return parser.parse_args()
 
 
+def check_positive_int(value):
```

Comment:
This is to make sure the value from CLI is a positive value. Otherwise, the logic in [line 202](https://github.com/codalab/codalab-worksheets/pull/2288/files#diff-c8428f7094434e5ab0590e15a34df0a6R202), needs to be `self.exit_number_jobs<=0`

Reply:
I don't think we need this since I think it's very unlikely someone will pass in a negative value in here.  We don't check this elsewhere.

Reply:
Removed.